packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -953,7 +954,7 @@ export default class TradeStore extends BaseStore {
         if (this.hovered_contract_type === contract_type) {
             this.addTickByProposal(response);
             setLimitOrderBarriers({
-                barriers: this.barriers,
+                barriers: this.root_store.portfolio.barriers,
```

Comment:
`this.root_store.portfolio.barriers` has been repeated in the code way too many times, perhaps there is a better way to use a shorter variable name instead of a store reference.

Reply:
sure, will declare a shorter variable for that

Reply:
since the this.root_store is a convention refers to the all stores of the core and it's using the same way in the all packages, I'm not sure how to shorten it. do you have any suggestion?